docs/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md

```diff
@@ -67,6 +67,18 @@ Configuring all streams in the input catalog to full refresh mode verifies that
 |||x|
 ||||
 
```

Comment:
beautiful 👍🏼

Reply:
I thought it should be there as an example, so thanks.